ci/README.md

```diff
@@ -0,0 +1,21 @@
+## Building the RPM locally
```

Comment:
For consistency, the other README.md left here (ci/deb-ppa-test-install/README.md) should be moved here as well